packages/jupyter/src/providers.tsx

```diff
@@ -14,10 +14,58 @@ import { useNotebookBase, useThebeConfig, useThebeCore, ThebeServerProvider } fr
 import type { Root } from 'mdast';
 import { useComputeOptions } from '@myst-theme/providers';
 
+function getThebeOptions(): CoreOptions {
+  const { thebe, binderUrl } = useComputeOptions();
+  const {
+    mathjaxUrl,
+    mathjaxConfig,
+    binder,
+    server,
+    kernelName,
+    sessionName,
+    disableSessionSaving,
+    local,
+  } = thebe;
+  const output: CoreOptions = { mathjaxUrl, mathjaxConfig };
+  if (binder) {
+    const useBinder = binder === true ? {} : binder;
+    output.binderOptions = {
+      binderUrl: useBinder.url ?? binderUrl,
+      ref: useBinder.ref,
+      repo: useBinder.repo,
+      repoProvider: useBinder.provider,
+    };
+  }
+  const useServer = local ?? server;
```

Comment:
I think we only choose one of these to use - preference goes to local config, when deploying we should strip `local` from the config.